samples/csharp_webapi/13.core-bot/Controllers/BotController.cs

```diff
@@ -58,6 +58,8 @@ static BotController()
             _dialog = new MainDialog(bookingRecognizer, bookingDialog, _loggerFactory.CreateLogger<MainDialog>());
         }
 
+        [HttpPost]
+        [HttpGet]
```

Comment:
Why do two separate attributes here?

Reply:
The styleguide insisted, it apparently has different rules for netCore vs netFx.